Source/Core/Core/PowerPC/JitCommon/JitBase.h

```diff
@@ -113,7 +115,7 @@ class JitBase : public CPUCoreBase
   JitOptions jo{};
   JitState js{};
 
-  JitBase();
+  explicit JitBase(std::size_t code_buffer_size);
```

Comment:
I wonder why is this configurable? The required size only depends on the maximum block size, not on the target plattform. So I like the idea to move the code_buffer to JitBase, but I see no reason to pick different sizes for different plattforms.

Reply:
Sure, I can remove that, I only did it this way to keep it like what the JITs were already doing, I'll update this to get rid of that; it'll also let us get rid of a separate [variable](https://github.com/dolphin-emu/dolphin/blob/master/Source/Core/Core/PowerPC/PPCAnalyst.cpp#L38) doing the same thing in PPCAnalyst as well.